telegram/bot/db.go

```diff
@@ -0,0 +1,96 @@
+package bot
+
+import (
+	"github.com/boltdb/bolt"
```

Comment:
It seems, we already use Redis as storage. Is it possible to store userIds in Redis instead of Bolt?